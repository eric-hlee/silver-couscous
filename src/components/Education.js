import React from 'react'
import { FormattedMessage } from 'react-intl'
import Card from 'react-bootstrap/Card'

export default function Education () {
  return (
    <Card>
      <Card.Link href="http://www.queensu.ca/" target="_blank">
        <Card.Img variant="top" src={require('../assets/images/queensu.png')} />
      </Card.Link>
      <Card.Body>
        <Card.Title>
          <FormattedMessage
            id="education.queensu"
            defaultMessage="Queen's University"
          />
        </Card.Title>
        <Card.Text>
          <strong>
            <FormattedMessage
              id="education.degree"
              defaultMessage="Bachelor of Computing (Honours)"
            />
          </strong>
          <br />
          <FormattedMessage
            id="education.program"
            defaultMessage="Specialization in Software Design with Professional Internship"
          />
          <br />
          <FormattedMessage
            id="education.deans_list"
            defaultMessage="Deanâ€™s Honour List 2017-18"
          />
        </Card.Text>
      </Card.Body>
    </Card>
  )
}
