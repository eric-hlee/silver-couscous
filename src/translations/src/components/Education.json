[{"id":"education.queensu","defaultMessage":"Queen's University","file":"src/components/Education.js","start":{"line":13,"column":5},"end":{"line":16,"column":7}},{"id":"education.degree","defaultMessage":"Bachelor of Computing (Honours)","file":"src/components/Education.js","start":{"line":20,"column":6},"end":{"line":23,"column":8}},{"id":"education.program","defaultMessage":"Specialization in Software Design with Professional Internship","file":"src/components/Education.js","start":{"line":26,"column":5},"end":{"line":29,"column":7}},{"id":"education.deans_list","defaultMessage":"Deanâ€™s Honour List 2017-18","file":"src/components/Education.js","start":{"line":31,"column":5},"end":{"line":34,"column":7}}]
