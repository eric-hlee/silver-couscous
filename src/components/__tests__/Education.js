import React from "react";
import { shallow } from "enzyme";
import Education from "../Education";

describe("<Education />", () => {
	it("renders correctly", () => {
		const wrapper = shallow(<Education />);
		expect(wrapper).toMatchSnapshot();
	});
});
