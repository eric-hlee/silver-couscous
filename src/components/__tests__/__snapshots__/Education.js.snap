// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Education /> renders correctly 1`] = `
<Card
  body={false}
>
  <CardLink
    href="http://www.queensu.ca/"
    rel="noopener"
    target="_blank"
  >
    <CardImg
      src="test-file-stub"
      variant="top"
    />
  </CardLink>
  <CardBody>
    <CardTitle>
      <FormattedMessage
        defaultMessage="Queen's University"
        id="education.queensu"
      />
    </CardTitle>
    <CardText>
      <strong>
        <FormattedMessage
          defaultMessage="Bachelor of Computing (Honours)"
          id="education.degree"
        />
      </strong>
      <br />
      <FormattedMessage
        defaultMessage="Specialization in Software Design with Professional Internship"
        id="education.program"
      />
      <br />
      <FormattedMessage
        defaultMessage="Deanâ€™s Honour List 2017-18"
        id="education.deans_list"
      />
    </CardText>
  </CardBody>
</Card>
`;
